agent/tool-scripts/base-tool

```diff
@@ -579,8 +579,10 @@ install)
 	function check_required_rpm {
 		local _tpn="${1}"
 		local _tpv="${2}"
-		local _installed_rpm="$(require-rpm "${_tpn}" "${_tpv}")"
-		local rc=${?}
+		local _installed_rpm
+		local rc
```

Comment:
Since we seem to have the convention of marking local variables with a leading underscore, it would probably be best to apply that to `rc` as well.  (I realize that you're not responsible for this lack, but, since you're here....)